.rubocop_todo.yml

```diff
@@ -36,11 +36,6 @@ Layout/AlignParameters:
 Layout/CaseIndentation:
   Enabled: false
 
-# Offense count: 28
-# Cop supports --auto-correct.
-Layout/ClosingParenthesisIndentation:
-  Enabled: false
```

Comment:
tbh i'm not a big fan of this cop's changes, some of them look good but others push parenthesis to odd locations.